src/node/index.js

```diff
@@ -551,12 +551,13 @@ Request.prototype._redirect = function (res) {
   _initHeaders(this);
 
   // redirect
+  this.res = res;
```

Comment:
@titanism This is the main change I'm not super certain about. Without doing this, a couple of the tests around `.pipe()` fail because there's no `this.res` on the response. I don't know if this assignment has the potential to screw anything else up.